src/gcodeExport.h

```diff
@@ -247,6 +248,11 @@ class GCodeExport : public NoCopy
     void writeTypeComment(PrintFeatureType type);
 
     /*!
+     * Write an M82 (absolute) or M83 (relative) depending on what extruder mode is in use
+     */
+    void writeExtruderMode();
```

Comment:
Suggested: `writeExtrusionMode`, i.e. sion instead of der.

Again kinda nit picking. Not a required change for this PR to be merged.